dfusion_rust_core/src/models/standing_order.rs

```diff
@@ -69,35 +89,205 @@ impl From<mongodb::ordered::OrderedDocument> for StandingOrder {
     }
 }
 
+impl From<&Arc<Log>> for StandingOrder {
+    fn from(log: &Arc<Log>) -> Self {
+        let mut bytes: Vec<u8> = log.data.0.clone();
+        println!("Parsing StandingOrder from bytes. {} bytes. {:?}", bytes.len(), bytes);
```

Comment:
can we make this an `info!(logger, ...)` instead of `println!`?

Reply:
Done!, I had to add the lib too.